configuration.php

```diff
@@ -259,6 +259,14 @@
             . '&view=minimal&limit=15&sort=host.state.severity desc',
             150
         );
+    } else {
+        /*
+        * Available navigation items
+        */
+        $this->provideNavigationItem('icingadb-host-action', $this->translate('Host Action (Icinga DB)'));
+        $this->provideNavigationItem('icingadb-service-action', $this->translate(
+            'Service Action (Icinga DB)'
+        ));
```

Comment:
Please use e.g. `$this->translate('Host Action') . ' (Icinga DB)'`. The latter doesn't need localization and this avoids duplicates in the message catalogs.